src/cpp/src/lora_adapter.cpp

```diff
@@ -139,18 +134,30 @@ ConstantMap read_safetensors(const std::filesystem::path& filename) {
 
         OPENVINO_ASSERT(
             ov::shape_size(shape) <= tensor.end_offset_bytes - tensor.begin_offset_bytes,
-            "Tensor shape ", ov::shape_size(shape), " for tensor \"", name, "\" from Safetensors file \"", filename, "\" doesn't match the expected tensor size ",
+            "Tensor shape ", ov::shape_size(shape), " for tensor \"", name, "\" from Safetensors file \"", "safetensor", "\" doesn't match the expected tensor size ",
```

Comment:
Please don't lose a better diagnostics for a file due to the code unification. Most users will still use adapters from files, so this message becomes less useful for them.